scenarios/contact-menu.json

```diff
@@ -68,6 +68,50 @@
       "url": "{url}/civicrm/group?reset=1",
       "onReadyScript": "contact/manage-groups.js"
     },
+    {
+      "label": "Manage Groups - Edit Group settings modal",
```

Comment:
why are certain words capitalized and certain not? sometimes it's "Modal" and sometimes "modal", for example. In this case it's "Group" instead of "group". Is there a pattern behind it?

Reply:
No there is not any pattern for this. Do you prefer to use any naming pattern? Should it be Camelcase or normal Capital Case ?